adafruit_ads1x15/ads1x15.py

```diff
@@ -69,66 +71,73 @@ def __init__(
         i2c: I2C,
         gain: float = 1,
         data_rate: Optional[int] = None,
-        mode: Mode = Mode.SINGLE,
+        mode: int = Mode.SINGLE,
         address: int = _ADS1X15_DEFAULT_ADDRESS,
     ):
         # pylint: disable=too-many-arguments
         self._last_pin_read = None
         self.buf = bytearray(3)
-        self._data_rate = self._gain = self._mode = None
+        self._gain: float
```

Comment:
Can you remove these annotations?  Since the CircuitPython parser sheds them, I'm worried this might look like a regular statement, so it will look like it hasn't been defined yet.  This goes for `self._data_rate` and `self._mode`.

Reply:
You could test this general premise of type annotating something like this, though, in a CircuitPython driver.  Let me know if you can and if it works!

Reply:
Should I just remove them or initialize them to something and let the property setters set them again based on their logic?
```
        self._gain = gain
        self.gain = gain
        self._data_rate = self._data_rate_default() if data_rate is None else data_rate
        self.data_rate = self._data_rate_default() if data_rate is None else data_rate
        self._mode = mode
        self.mode = mode
```

Reply:
See next review comment!